src/collaborative/ot/ot_specific.ts

```diff
@@ -141,3 +151,24 @@ function freezeTransformation(
   }
   return quantity > 0 ? { ...cmd, quantity } : undefined;
 }
+
+/**
+ * Cancel CREATE_FILTER_TABLE and ADD_MERGE commands if they overlap a filter
+ */
+function createTableTransformation(
```

Comment:
it could be the exact same transformation as `mergeTransformation`, right ? (should be renamed if it's reused)
It wouldn't drop the entire command (just the overlapping zoes), but it's ok from a functional point of view I think

Reply:
It's different because we don't create a table for each target in `CREATE_FILTER_TABLE `, but we merge the targets together to create a single table (if the targets are continuous, else the allowdispatch prevent everythin).
So it make sense here to drop the whole command if a target overlap a merge/another table